pom.xml

```
@@ -101,6 +101,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
+      <version>3.4.1</version>
```

Needs to require the git client plugin version that fixes the null pointer exception in `UnsupportedCommand`.
```suggestion
      <version>3.4.2</version>
```